src/Makefile

```diff
@@ -47,10 +47,10 @@ CCOPT= -O2 -fomit-frame-pointer
 # x86/x64 only: For GCC 4.2 or higher and if you don't intend to distribute
 # the binaries to a different machine you could also use: -march=native
 #
-CCOPT_x86= -march=i686 -msse -msse2 -mfpmath=sse
+CCOPT_x86= -march=i686 -msse -msse2 -msse4.2 -mfpmath=sse
 CCOPT_x64=
 CCOPT_arm=
-CCOPT_arm64=
+CCOPT_arm64= -march=armv8-a+crc
```

Comment:
Specifically, this; the flags should not be set for all sources, only for lj_str_hash.c so that everything else is built to run on CPUs without SSE4.2.  Also, it's curious that CCOPT_x64 is not set; I've not tested this but it appears that 64-bit will not use sse4.2.

Reply:
Fair enough - mind having another look at the latest version of this patch? Now, `jit.crc32()` will always attempt to detect CRC32 support, and `lj_str_hash.c` will be the only source file compiled with CRC32 support.